styles/uswds-dependencies.scss

```diff
@@ -0,0 +1,9 @@
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-general';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-typography';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-spacing';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-color';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-utilities';
+@import 'settings.scss';
+@import 'node_modules/uswds/src/stylesheets/packages/_required.scss';
+
+@import 'node_modules/uswds/src/stylesheets/core/_notifications.scss';
```

Comment:
what's this one for? I noticed it's not included in the prototype PR

Reply:
debugging remnant! removed✅